spec/requests/users_spec.rb

```diff
@@ -33,7 +33,7 @@
 
     it 'Include correct placeholder text in response body' do
       get user_url(1)
-      expect(response.body).to include('show the user id details')
+      expect(response.body).to include('users details')
     end
   end
 end
```

Comment:
- [ ] `README.md` file: To make your documentation be more professional and descriptive to non technical users, I think it would be better if you update your `Getting started` section with all information and appropriate information needed by users such as `tools needed(Prerequisites)` in order to use your app locally, information of how `clone` your project, `Access` the cloned directory, `install` bundles, `create database`, `use/run` your application locally, `run tests`,... Right now, most of information are missing in your documentation. I **suggest you refer [this Gist file](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)** to help you how to proceed.

Reply:
done!